### PR TITLE
Creating functions for starting, stopping, and restarting a website.

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

File: Carbon.IIS/Functions/Install-CIisWebsite.ps1 [hunk 1]
```diff
@@ -98,7 +98,9 @@ function Install-CIisWebsite
         [bool] $ServerAutoStart,
 
         # Return a `Microsoft.Web.Administration.Site` object for the website.
-        [switch] $PassThru
+        [switch] $PassThru,
+
+        [TimeSpan] $Timeout = [TimeSpan]::New(0, 0, 30)
```
Comment (decoyjoe) [inline Carbon.IIS/Functions/Install-CIisWebsite.ps1:103]: Would it be simpler to use `New-TimeSpan -Seconds 30`?

File: Carbon.IIS/Functions/Restart-CIisWebsite.ps1 [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    The `Restart-CIisWebsite` restarts an IIS website. Pass the names of the websites to restart to the `Name`
+    parameter. You can also pipe website objects or website names.
+
+    The website is stopped then started. If stopping the website fails, the function does not attempt to start it. If
+    after 30 seconds, the website hasn't stopped, the function writes an error, and returns; it does not attempt to
+    start the website. Use the `Timeout` parameter to control how long to wait for the website to stop. The function
+    writes an error if the website doesn't stop or start.
+
+    .EXAMPLE
+    Restart-CIisWebsite -Name 'Defaul Website'
```
Comment (decoyjoe) [inline Carbon.IIS/Functions/Restart-CIisWebsite.ps1:19]: Typo: `Defaul`

**Review by decoyjoe — APPROVED** [review]

File: Carbon.IIS/Functions/Install-CIisWebsite.ps1 (hunk 1 above)
Comment (splatteredbits) [reply Carbon.IIS/Functions/Install-CIisWebsite.ps1:103]: Either works. If I never need to mock out the default timeout, I'll use `New-TimeSpan`. (I actually didn't know that cmdlet existed.)

File: Carbon.IIS/Functions/Restart-CIisWebsite.ps1 (hunk 2 above)
Comment (splatteredbits) [reply Carbon.IIS/Functions/Restart-CIisWebsite.ps1:19]: Will fix in a future PR.

**Pull request merged**